tests/addons/TA_CIM_Broken/app.manifest

```diff
@@ -0,0 +1,53 @@
+{
+  "schemaVersion": "1.0.0",
```

Comment:
Which version of Slim is used? Looks like an older version.

Reply:
In latest SLIM schema version should be 200 https://dev.splunk.com/enterprise/docs/releaseapps/packagingtoolkit/pkgtoolkitref/pkgtoolkitapp/

Reply:
we are not actually using slim I just used it to get the package ID so I didn't notice i used an old version fixed that now